src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppKeys.scala

```diff
@@ -105,10 +105,27 @@ trait SbtReactiveAppKeys {
   val akkaManagementEndpointName = SettingKey[String]("rp-akka-management-endpoint-name")
 
   /**
-   * Defines packages to install on top of the base alpine image.
+   * Defines packages to install on top of the base docker image.
    */
+  val userPackages = SettingKey[Seq[String]]("rp-user-packages")
```

Comment:
Linux packaging is not a specific feature of Akka or Lagom, so I think we don't need to add these.
Instead I think we should have a Boolean setting that opts out of current`/sbin/apk` calls. That would be enough for me to switch to RPM or apt-get based distro.

Reply:
It depends on the base image selected. If the user wishes to use a stock Fedora image, for instance, then OpenJDK needs to be installed separately. This functionality was already present in the `alpinePackages` settings, I just renamed the setting since it no longer refers to Alpine exclusively.